imagersite/media/relations.css

```diff
@@ -0,0 +1,6 @@
+svg .link
+{
+  stroke: steelblue;
+  stroke-opacity: .4;
+  fill: none;
+}
```

Comment:
This is a static file, not a media file.  It should be in a static directory rather than a directory where user-uploaded content will go.